### DISCUSSION TITLE
Size of supercell

### DISCUSSION THREAD

**Discussion by chejunwei2** [ttadano/alamode#67] [Q&A]
Dear Dr. Tadano,
I want to use ALAMODE to calculate thermal conductivity. The material I studied is La2Zr2O7 (a=b=c=10.79 Å), and its unit cell has 88 atoms. If I use supercell (e.g., 2×2×2) to calculate thermal conductivity, it will need a high calculation cost. So I wonder if I can use unit cell (i.e., 1×1×1) to calculate thermal conductivity. Thanks very much.
Best regards

**Comment by ttadano** [comment]
Yes, it is possible to compute thermal conductivity using the primitive cell. That would be reasonable if the primitive cell is sufficiently large.
